api-spec/STAC-extensions.yaml

```diff
@@ -761,35 +769,44 @@ components:
       properties:
         bbox:
           $ref: '#/components/schemas/bbox'
-    timeFilter:
```

Comment:
moved this down so bbox and intersects would be next to one another, and the 3 datetime-related ones would be also.